crawlers/reddit/source/utils/tools.py

```diff
@@ -0,0 +1,63 @@
+import json
+import requests
+import requests.auth
+from pathlib import Path
+from .log import logger
+
+config_path = (Path(__file__).absolute().parent.parent.parent/'config.json')
+data_path = (Path(__file__).absolute().parent.parent.parent/'data.json')
+
+def read_config():
+    with open(config_path, 'r') as f:
+        data = json.load(f)
+        return data
+
+def save_list(data):
+    with open(data_path, "w") as f:
+        json.dump(data, f)
+
+
+def read_list():
+    with open(config_path, 'r') as f:
+        data = json.load(f)
+        return data.get('keywords')
+
+
+def read_access_token():
+    with open(config_path, "r") as f:
+        data = json.load(f)
+        return data.get('access_token')
+
+
+def update_config(**kwargs):
+    '''Update config file
+    params must be specify'''
+    conf = read_config()
+
+    for key in kwargs.keys():
+        if key in conf.keys():
+            conf[key] = kwargs[key]
+
+    with open(config_path, "w") as f:
+        json.dump(conf, f)
+
+
+def get_access_token(username, password, client_id, client_secret):
```

Comment:
What do you think about creating a separate class for login and working with access token?